releases/release-4.0.16.md

```diff
@@ -0,0 +1,115 @@
+    - Fix incorrect by-instance gRPC average duration [#11326](https://github.com/tikv/tikv/pull/11326)
+    - Fix CDC panic due to missing downstream [#11135](https://github.com/tikv/tikv/pull/11135)
+    - Fix channel full could break the raft connection [#11069](https://github.com/tikv/tikv/pull/11069)
+    - Fix Max/Min bug when comparing signed and unsigned int64 [#10616](https://github.com/tikv/tikv/pull/10616)
+    - Fix frequent CDC incremental scan retry due to `Congest` error [#11089](https://github.com/tikv/tikv/pull/11089)
+
++ PD
+
+    - Fix panic issue after TiKV node scales in [#4378](https://github.com/tikv/pd/pull/4378)
+    - Fix the issue that PD may not elect leader as soon as leader step down [#4219](https://github.com/tikv/pd/pull/4219)
+    - `evict-leader-scheduler` supports schedule the regions with unhealthy peers [#4133](https://github.com/tikv/pd/pull/4133)
+
++ TiFlash
+
+    - Fix the issue that comparison between Decimal may cause overflow and report `Can't compare`
+    - Fix the issue that TiFlash fails to start up under platform without library `nsl`
+
++ Tools
+
+    + TiCDC
+
+        - Fix changefeed checkpoint lag negative value error [#3532](https://github.com/pingcap/ticdc/pull/3532)
+        - Fix OOM in container environments [#3440](https://github.com/pingcap/ticdc/pull/3440)
+        - Fix TiCDC could meet replication interruption when multiple TiKVs crash or forcing restart [#3290](https://github.com/pingcap/ticdc/pull/3290)
+        - Fix memory leak after processing DDLs [#3274](https://github.com/pingcap/ticdc/pull/3274)
+        - Fix changefeed does not fast fail when occur ErrGCTTLExceeded error [#3134](https://github.com/pingcap/ticdc/pull/3134)
+        - Fix fallback resolvedTs event will block the progress of resolve lock when occur region merging [#3099](https://github.com/pingcap/ticdc/pull/3099)
+        - Close gPRC stream and re-create it when meeting `ErrPrewriteNotMatch` to avoid duplicated request error [#3094](https://github.com/pingcap/ticdc/pull/3094)
+        - Fix kafka sink can not send message due to constraint by `max-message-size` option [#3046](https://github.com/pingcap/ticdc/pull/3046)
+        - Fix tikv_cdc_min_resolved_ts_no_change_for_1m keep firing when there is no changefeed [#3023](https://github.com/pingcap/ticdc/pull/3023)
+        - Fix possible deadlocking when Kafka producer reports an error [#3015](https://github.com/pingcap/ticdc/pull/3015)
+        - Fix dml is not replicated after adding partition in partition table without valid index [#2863](https://github.com/pingcap/ticdc/pull/2863)
+        - Fix memory leak which may happen in create new changefeed [#2623](https://github.com/pingcap/ticdc/pull/2623)
+        - Set config.Metadata.Timeout correctly to prevent stuck data synchronization [#3669](https://github.com/pingcap/ticdc/pull/3669)
+
+## Critical bug not obtained through tools
+
++ TiDB
```

Comment:
```suggestion
+ TiDB

    - expression: fix wrong flen when cast decimal to string. [#30012](https://github.com/pingcap/tidb/pull/30012)

    - planner: change redundantSchema to fullSchema to correctly handle natural and "using" joins. [#30037](https://github.com/pingcap/tidb/pull/30037)

    - planner: fix topn wrongly pushed to index scan side when it's a prefix index [#29796](https://github.com/pingcap/tidb/pull/29796)
```

Reply:
LGTM

Reply:
@tiancaiamao Where should we put them? Under Bug fixes? Could you please help add the Chinese version? Thanks.